client/src/legacy/GridField.js

```diff
@@ -569,6 +569,14 @@ $.entwine('ss', function($) {
       } else {
         this._super(e);
       }
+
+      const toastNotificationMessage = $(this).hasClass('action--archive')
+      ? ss.i18n._t('Admin.ARCHIVE_CONFIRM_MESSAGE', 'Archived')
+      : ss.i18n._t('Admin.DELETE_CONFIRM_MESSAGE', 'Deleted');
+
+      if (!$(this).hasClass('action--delete')) {
```

Comment:
`action--delete` also exist on `Delete` button in GridField item itself. Check this to avoid invoking message on this event. Otherwise, we have two messages.

Reply:
I'm not getting any message at all when clicking the "delete" button on an item in the registry modeladmin
![image](https://github.com/silverstripe/silverstripe-admin/assets/36352093/f4669151-c5b0-460c-b1fe-ef135a49ff61)

Reply:
Updated